Database/Corrections/QuestieQuestFixes.lua

```diff
@@ -230,6 +231,17 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.sourceItemId] = 18746,
             [QuestieDB.questKeys.zoneOrSort] = 1637,
         },
+        [7669] = { --#1449
+            [QuestieDB.questKeys.name] = "Again Into the Great Ossuary",
+            [QuestieDB.questKeys.startedBy] = {{928,},nil,nil,},            -- Quest is started by Lord Grayson Shadowbreaker
+            [QuestieDB.questKeys.finishedBy] = {{928,},nil,nil,},           --       & ended*
+            [QuestieDB.questKeys.requiredLevel] = 58,
+            [QuestieDB.questKeys.questLevel] = 60,
+            [QuestieDB.questKeys.requiredRaces] = 178,                      -- Any race can take on quest
```

Comment:
If you have a look at [the wiki](https://github.com/AeroScripts/QuestieDev/wiki/Database-Structure) you'll see that the value for all races is 255 🙂 178 is Horde, but I haven't had a look at the quest :sweat_smile:

Reply:
Since this is a Shaman quest nevermind the previous comment...